external/fixedWidth/src/test/scala/za/co/absa/enceladus/fixedWidth/FixedWidthRelationSuite.scala

```diff
@@ -17,6 +17,6 @@ package za.co.absa.enceladus.fixedWidth
 
 import org.scalatest.FunSuite
 
-class FormatSuite extends FunSuite {
+class FixedWidthRelationSuite extends FunSuite {
 
```

Comment:
Empty test suite? Should we remove it? Or add a `ToDo` so we won't forget to add tests for handling fixed width text files.

Reply:
IIRC I had created the empty test, because something somewhere complained about the folder not existing. And before this commit I noticed that there was a wrong class name within the "test"
I'll create a ticket for the test to be implemented, but would consider in a low prio,